src/content/ocaml/colophon.tex

```diff
@@ -0,0 +1 @@
+OCaml code translation was done by Arulselvan Madhavan and reviewed by Marcello Seri and Anton Kochkov.
```

Comment:
@ArulselvanMadhavan @mseri @XVilka 
I've extracted the colophon into a separate page. Would you like me to hyperlink your names? Where should they point to?

Reply:
Oh, nice. Please link `www.mseri.me`

Reply:
I've added the github urls for Arul and Anton. We can fix it in post :)